frontend-public/src/components/dashboard/mine/variances/Variances.js

```diff
@@ -49,6 +68,80 @@ export class Variances extends Component {
     this.props.fetchVarianceStatusOptions();
   }
 
+  handlAddDocuments = (files, varianceGuid) => {
```

Comment:
Should be `handleAddDocuments` 😃

Reply:
Put that there just to see if you're paying attention haha